src/olympia/addons/tests/test_views.py

```diff
@@ -2907,6 +2905,31 @@ def test_basic(self):
             u'misc': False,
             u'id': 1,
             u'application': u'firefox',
+            u'description': None,
+            u'type': u'extension',
+            u'slug': u'feeds-news-blogging'
+        }
+
+    def test_with_description(self):
+        # StaticCategories are immutable so have to hack around.
```

Comment:
This is not necessarily a "hack", it's just our way of setting stuff on them which exists thanks to pythons dynamic nature. I'd comment this like `StaticCategory is immutable, so avoid calling it's __setattr__ directly.` I think this is more correct.